version/1/4/it/code_of_conduct.md

```diff
@@ -0,0 +1,47 @@
+# Patto di Condotta dei Collaboratori
```

Comment:
I am not really fond of the word `Patto`, I'd stay closer to the original and call it `Codice`, also because the "literature" is already going in that direction (ex. [Italian version of the Berlin Code of Conduct](http://berlincodeofconduct.org/it/))